brick/__brick__/lib/counter/counter.dart

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2022, Very Good Ventures
+// Copyright (c) {{current_year}}, Very Good Ventures
```

Comment:
We shouldn't need to modify the brick files manually. They should be updated by the bot when the PR is merged

Reply:
Reverted the changes.